crates/css_ast/src/values/fonts/impls.rs

```diff
@@ -41,6 +41,7 @@ mod tests {
 		assert_parse!(FontStyleStyleValue, "normal");
 		assert_parse!(FontStyleStyleValue, "oblique 45deg");
 		assert_parse_error!(FontStyleStyleValue, "oblique 45px");
+		assert_parse_error!(FontStyleStyleValue, "oblique 91deg");
```

Comment:
Nit: Something I overlooked in the previous PR, can we add a `fn test_errors()` just for the error conditions, that way it's easier to read the two. Maybe also a comment above each parse error explaining why.